src/Hubbup.Web/bundleconfig.json

```diff
@@ -0,0 +1,24 @@
+﻿// Configure bundling and minification for the project.
```

Comment:
It's a file that you have to opt-in to using if you install the bundler/minifier... but I don't know that people do. I don't 😄